README.md

```diff
@@ -3,8 +3,29 @@
 
 Boilerplate project for StockTicker code exercise using React.
 
-This project has a single App Component which subscribes to an StockTicker event provided by the StockTicker library located in `/public/lib/TickerInterview`. This subscription receives a Tick object at random intervals with a stock symbol and price information. Currently, these objects are being logged to the console as they are received.
+## Quick Start
 
-Your objective is to flesh out a simple Single Page App that displays these stock prices. A simple solution will show all events in an expanding list, while a more robust solution will keep a single entry for each stock and update it as events come in. Bonus points for adding visual effects to stocks as they change, such as colors to indicate if a stock increased or decreased, or moving relative positions to indicate what event happened most recently.
+1. Clone this repository
+2. Run `npm install`
+3. Run `npm start` to start the dev server
+4. Navigate to <http://localhost:3000/>
 
-Run `npm start` for a dev server. Navigate to http://localhost:3000/. The app will automatically reload if you change any of the source files.
+The app will automatically reload if you change any of the source files.
+
+## Overview
+
+This project has a single App Component which subscribes to a `stockTicker$` Observable provided by the stock ticker service. This subscription receives a `Tick` object at random intervals with a stock symbol and price information. These objects are logged to the console as they are received.
+
+## Assignment
```

Comment:
Are we thinking of giving this out asynchronously? I don't know if we want to preview what the user will have to do in the interview because I like having the candidate refactor code as we request more and more "features" on the stockticker

Reply:
The previous version of the README had a rough outline of the assignment, but for live coding interviews, I agree that it is nice to have the candidate refactor as complexity is added. I'll remove the "Assignment" section for now.